livekit-plugins/livekit-plugins-rime/livekit/plugins/rime/tts.py

```diff
@@ -66,16 +66,14 @@ class _Mistv2Options:
     phonemize_between_brackets: NotGivenOr[bool] = NOT_GIVEN
 
 
-DEFAULT_API_URL = "https://users.rime.ai/v1/rime-tts"
```

Comment:
wdyt of keeping this as a const at the top of file?

Reply:
I made it a default value in the init - so it keeps the same format as setting the default model